lib/addons/amp/index.ts

```diff
@@ -97,10 +97,7 @@ export class AmpAddOn implements ClusterAddOn {
             tags: defaultProps.workspaceTag
         };
 
-        if (this.ampAddOnProps.prometheusRemoteWriteURL) {
-            remoteWriteEndpoint = this.ampAddOnProps.prometheusRemoteWriteURL;
-        }
-        else {
+        if (typeof(this.ampAddOnProps.prometheusRemoteWriteURL) == 'undefined') {
```

Comment:
The previous logic checks both for undefined and null. This only checks for undefined. I think we need to handle both. Difference:

Null is passed like this:
```
{
    prometheusRemoteWriteURL: null
}
```

Undefined is passed by not specifying property at all.

Reply:
Fixed.